src/components/Projects/FilterCheckboxItem.tsx

```diff
@@ -0,0 +1,45 @@
+import { Trans } from '@lingui/macro'
+import { Checkbox } from 'antd'
+import { ThemeContext } from 'contexts/themeContext'
+import { CSSProperties, useContext } from 'react'
+
+export default function FilterCheckboxItem({
+  label,
+  checked,
+  onChange,
+  disabled,
+}: {
+  label: string
+  checked: boolean
+  onChange: Function
```

Comment:
FWIW, I don't like typing functions as just a `Function`, especially when it's not _just_ a function; it has a parameter. I would prefer typing this as `(checked: boolean) => void`, that way callers know what to expect from this prop.

Reply:
Yep good call!